src/Fantomas.Tests/UtilsTests.fs

```diff
@@ -111,4 +113,67 @@ SetupTesting.generateSetupScript __SOURCE_DIRECTORY__
 #load "__setup__.fsx"
 #endif
 """
-    |> mergeAndCompare a b
+        |> mergeAndCompare a b
+
+module List =
+    [<Test>]
+    let ``When input is empty`` () =
+        let property (p: bool) : bool =
+            let before, after = List.partitionWhile (fun _ _ -> p) []
+            before = [] && after = []
+
+        Check.QuickThrowOnFailure(property true)
+        Check.QuickThrowOnFailure(property false)
+
+    [<Test>]
+    let ``When predicate always returns false`` () =
```

Comment:
Could you lower-case the unit test name? Most of the unit tests start with a lower-case, see https://github.com/fsprojects/fantomas/blob/master/CONTRIBUTING.md#guidelines.

Reply:
Woops, this file happens to have tests with upper-case but I've switched mine to lower. I'll do a separate PR after this one to lower-case any other stragglers.